contact.html

```diff
@@ -114,7 +114,18 @@ <h4 class="translate" aria-label = "लाभार्थियों के ल
           </p>
         </div>
         <div class="contact-right">
-          <h4 class="translate" aria-label = "दाताओं के लिए">For Donors</h4>
+          <h3 class="translate" aria-label = "दाताओं के लिए">For Donors</h3>
+          <h4 class = "translate"> Donation Details</h4>
+          <div>
+            <p class="translate" aria-label="बैंक: भारतीय स्टेट बैंक, जसूर">Bank: State Bank of India, Jassur</p>
+            <p class="translate" aria-label="खाते का नाम: रणजीत बख्शी जनकल्याण फाउंडेशन">Account Name: Ranjeet Bakshi Jankalyan Foundation</p>
+            <p class="translate" aria-label="खाता क्रमांक: 40354212801">Account Number: 40354212801</p>
+            <p class="translate" aria-label="आईएफएससी कोड: SBIN0002369">IFSC Code: SBIN0002369</p>
+            <p>Gpay: 9418151074</p>
+            <p>UPI: 9418151074@upi</p>
+          </div>
+          <h4 class = "translate">Get in touch</h4>
+          <br /> <!-- I hate this but better than adding CSS code right? -->
```

Comment:
:P that's right

Reply:
Thanks to your comment, I noticed that the above line doesn't have the translation bit. Added now lol.